docs/visual-basic/programming-guide/concepts/linq/how-to-use-annotation-trees-to-transform-linq-to-xml-trees-in-an-xslt-style.md

```diff
@@ -154,30 +154,32 @@ After Transform
   
 - The pseudo code for the function is quite simple:  
   
+```vb  
```

Comment:
This shouldn't be in a language block. It would be better with `>`. For example:
> The function takes....
> 
> If an element has an XElement annotation, then return a new XElement.
> - The name...
> - All attributes
...
> If an element is not annotated, then return a new XElement
> - The name...

Reply:
@rpetrusha Good suggestion Ron.